server/routes/quiz.route.ts

```diff
@@ -0,0 +1,41 @@
+const express = require("express");
+const router = express.Router();
+const { Quiz: QuizDB } = require("../models/quiz.model");
+const {
+  QuizCategory: quizCategoryDB,
+} = require("../models/quiz.category.model");
+router.route("/category").post(async (req: any, res: any) => {
```

Comment:
- Don't use `any` for `req` and `res` instead use `express.Request` and `express.Response`